spec/services/zendesk_sender_spec.rb

```diff
@@ -33,14 +34,17 @@
 
     it 'calls ZendeskAPI::Ticket.create!' do
       zen_call
-      expect(ZendeskAPI::Ticket).to have_received(:create!).with(ZENDESK_CLIENT, hash_including(bug_report_zendesk_payload))
+      expect(ZendeskAPI::Ticket)
+        .to have_received(:create!)
+        .with(ZENDESK_CLIENT, hash_including(bug_report_zendesk_payload))
     end
 
-    context 'with a reporter email' do
-      let(:ticket_payload) do
+    context 'with a valid ticket and reporter email' do
+      let(:bug_report_ticket_payload) do
```

Comment:
Is this the same bug_report_ticket_payload as the one above? Are you able to reuse it somehow?

Reply:
This is code that was already in this test file that I've just tweaked. The difference between them is that the first instance of the payload has `nil` for the email address, while this one has is populated. I assume this is to make sure that it can run with and without an email without raising an error.

@jrmhaig git blame shows you as the original author of most of this file, am I understanding the purpose of that right?